### PR TITLE
Added some things

### PR DESCRIPTION
**Pull request by efefury**
Added experiment timer todo overlay 
Added total xp if player is above cata 50 in /pv
Added daily powder/catacombs runs to todo overlay
Added a way to include kismet feather to profit calculator

### CODE REVIEW

**Review by Lulonaut — APPROVED** [review]
lgtm

**Review by Lulonaut — DISMISSED** [review]
tested all the features
lgtm

File: src/main/java/io/github/moulberry/notenoughupdates/profileviewer/GuiProfileViewer.java [hunk 1]
```diff
@@ -1369,15 +1370,20 @@ private void renderXpBar(String skillName, ItemStack stack, int x, int y, int xS
...
                     levelStr = EnumChatFormatting.GOLD + "MAXED!";
+                    totalXpStr = EnumChatFormatting.GRAY + "Total XP: " +EnumChatFormatting.DARK_PURPLE+ Utils.formatNumberWithDots((long) levelObj.totalXp);
                 } else {
                     int maxXp = (int) levelObj.maxXpForLevel;
                     levelStr = EnumChatFormatting.DARK_PURPLE + shortNumberFormat(Math.round((level % 1) * maxXp),
                             0) + "/" + shortNumberFormat(maxXp, 0);
                 }
-
-                tooltipToDisplay = Utils.createList(levelStr);
+                if(totalXpStr != null){
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/profileviewer/GuiProfileViewer.java:1382]: needs a space after the if

File: src/main/java/io/github/moulberry/notenoughupdates/profileviewer/GuiProfileViewer.java [hunk 2]
```diff
@@ -1369,15 +1370,20 @@ private void renderXpBar(String skillName, ItemStack stack, int x, int y, int xS
...
+                    tooltipToDisplay = Utils.createList(levelStr, totalXpStr);
+                }else {
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/profileviewer/GuiProfileViewer.java:1384]: needs a space before the else

File: src/main/java/io/github/moulberry/notenoughupdates/profileviewer/GuiProfileViewer.java [hunk 3]
```diff
@@ -1369,15 +1370,20 @@ private void renderXpBar(String skillName, ItemStack stack, int x, int y, int xS
         if (mouseX > x && mouseX < x + 120) {
             if (mouseY > y - 4 && mouseY < y + 13) {
                 String levelStr;
+                String totalXpStr  = null;
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/profileviewer/GuiProfileViewer.java:1373]: only 1 space before the `=`

File: src/main/java/io/github/moulberry/notenoughupdates/profileviewer/GuiProfileViewer.java [hunk 4]
```diff
@@ -1369,15 +1370,20 @@ private void renderXpBar(String skillName, ItemStack stack, int x, int y, int xS
...
                 if (levelObj.maxed) {
                     levelStr = EnumChatFormatting.GOLD + "MAXED!";
+                    totalXpStr = EnumChatFormatting.GRAY + "Total XP: " +EnumChatFormatting.DARK_PURPLE+ Utils.formatNumberWithDots((long) levelObj.totalXp);
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/profileviewer/GuiProfileViewer.java:1376]: unequal spacing on `+`

**Review by IRONM00N — CHANGES_REQUESTED** [review]
:trollface:

**Review by IRONM00N — APPROVED** [review]

**Review by Lulonaut — APPROVED** [review]
I am being forced to accept this

**Review by NopoTheGamer — APPROVED** [review]
looks good

**Pull request merged**